SS14.Labeller/Repository/RepositoryBase.cs

```diff
@@ -9,7 +9,7 @@ protected DbConnection OpenConnection()
     {
         var connectionString = configuration.GetConnectionString("Default") 
                                ?? "Data Source=Application.db";
```

Comment:
don't default to "Data Source=..." in this case. Since that isn't a valid connection string for postgres. Throw here instead.